### PR TITLE
bower: add bower.json

### PR DESCRIPTION
**Pull request by davidchambers**
See #88

I used Bower's guide to [creating packages][1] as a reference.

Could you review this change, @ChetHarrison?

[1]: http://bower.io/docs/creating-packages/

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "/coverage/",
+    "/node_modules/",
+    "/scripts/",
+    "/test/",
+    "/.*",
+    "/CONTRIBUTING.md",
+    "/Makefile",
+    "/circle.yml",
+    "/package.json"
+  ]
```
Comment (davidchambers) [inline bower.json]: Given that I've been unable to determine which [paths Bower ignores by default][1], I've listed all the applicable paths explicitly (including paths such as `"/bower_components/"` which are surely ignored regardless).


[1]: http://stackoverflow.com/q/22598184/312785

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "xyz": "0.5.x"
   },
   "files": [
+    "/LICENSE",
```
Comment (davidchambers) [inline package.json]: This is included in the npm package regardless, but it's nice to be explicit.

**Comment by ChetHarrison** [comment]
sure. just got back from lunch

On Sep 18, 2015, at 1:43 PM, David Chambers <notifications@github.com> wrote:

See #88 <https://github.com/plaid/sanctuary/issues/88>
I used Bower's guide to creating packages <http://bower.io/docs/creating-packages/> as a reference.

Could you review this change, @ChetHarrison <https://github.com/ChetHarrison>?

You can view, comment on, or merge this pull request online at:

  https://github.com/plaid/sanctuary/pull/91 <https://github.com/plaid/sanctuary/pull/91>
Commit Summary

bower: add bower.json
File Changes

A bower.json <https://github.com/plaid/sanctuary/pull/91/files#diff-0> (41)
M package.json <https://github.com/plaid/sanctuary/pull/91/files#diff-1> (1)
Patch Links:

https://github.com/plaid/sanctuary/pull/91.patch <https://github.com/plaid/sanctuary/pull/91.patch>
https://github.com/plaid/sanctuary/pull/91.diff <https://github.com/plaid/sanctuary/pull/91.diff>

**Comment by ChetHarrison** [comment]
looks good to me. Once I figure out the UMD wrapper we can add `amd` to the `moduleTypes`

**Comment by ChetHarrison** [comment]
Thanks!

**Pull request merged**